app/src/main/res/layout/activity_maps.xml

```diff
@@ -1,9 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
-<fragment xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:map="http://schemas.android.com/apk/res-auto"
+
+<RelativeLayout android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/map"
-    android:name="com.google.android.gms.maps.SupportMapFragment"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MapsActivity" />
\ No newline at end of file
+    tools:context=".MapsActivity">
+
+        <fragment xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
i don't know how i feel about hardcoding a fragment into the activity's xml. i'd say that it's ok as long as you are confident that this activity will contain this fragment at all times

Reply:
This is the map fragment (which is always present). Also Android Studio initializes the xml with this whenever a new MapsActivity is added, so I think it may be standard.